docs/community/glossary.md

```diff
@@ -0,0 +1,15 @@
+| napari term        | Description                                                                                                                                                                                                                                                                                                                                 |
+|--------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| channel            | A channel is a grayscale image. In fluorescence microscopy,  each channel corresponds to a range of wavelengths. An RGB image from digital camera consists of 3 channels - red, green, blue.                                                                                                                                                |
```

Comment:
* I think it would be good to link to more detailed definitions/descriptions for these terms. Here that might be the wikipedia page https://en.wikipedia.org/wiki/Channel_(digital_image)
* "A channel is a grayscale image" is not great phrasing, this is going to cause confusion for people new to image processing. I understand where you got this from, but I think we need a better way to explain it.
* We might also add a small note that sometimes people call these "bands" (I think this is relatively common for geoscience/satellite imagery people, and maybe in astronomy too?)

Reply:
> I think it would be good to link to more detailed definitions/descriptions for these terms.

Agreed, I think we should do that for most (if not all) general terms

> "A channel is a grayscale image" is not great phrasing

Yeah I kind of agree, even though it is technically correct.  Are there any fields where a `channel` **isn't** a wavelength? This is true for geospatial as well. We could say something like:

"A channel is an image acquired by a camera that captures a specific range of wavelengths. In fluorescence microscopy, common channels might be x, y, z (I don't know any), while common channels in a standard camera are red, green and blue"

Reply:
> We might also add a small note that sometimes people call these "bands"

Yes I thought we were considering having an "also known as" disambiguation column?

Reply:
I'm hesitant to use acquisition method or wavelength strictly as a way to define channel, because its definition is not really tied to specific devices. For example, a typical confocal uses PMT rather than camera for acquisition. Fluorescence lifetime acquisition is another example that we can have "channels" but not based on wavelength.
  
If we replace the first sentence to  "A channel is an image that each pixel only contains its intensity value (i.e. a grayscale image)..." would it sound clearer?

Reply:
> If we replace the first sentence to "A channel is an image that each pixel only contains its intensity value (i.e. a grayscale image)..." would it sound clearer?

I don't think that is much clearer, sorry

Reply:
> > If we replace the first sentence to "A channel is an image that each pixel only contains its intensity value (i.e. a grayscale image)..." would it sound clearer?
>
> I don't think that is much clearer, sorry

I do think it's better. I don't think we want to make it the explanation too domain specific. We mainly use the word channel in the `channel_axis` kwarg in `add_image` (which i think we might want to drop), "split_channels" and when  we talk about "multichannel" data in the docs. This often is in the context of data where you want to associate a different slice of dataset with a different colormap or different contrast limits/ rendering parameters. I'd say most often in optical imaging it is related to wavelengths, but in other modalities it need not be. For example in MRI you could think of  a T1 weighted vs a T2 weighted image as being in different "channels".

In my mind, I think of a "channel" as a dimension of data that are not spatial or temporal, and so looking at different values of the channel you're still looking at the same space and time (up to the limits of your acquisition system). So I'd say a 6D data set of a volumetric timeseries of a single sample would have two additional "channels" in it that could correspond to wavelengths, laser intensity etc.

What does @jni recommend we write here?

Reply:
> In my mind, I think of a "channel" as a dimension of data that are not spatial or temporal

I think this is getting at it a bit more clearly. 

> I don't think we want to make it the explanation too domain specific.

I'm happy for the first sentence to be non-domain specific, but after that I think we need to include examples that are domain specific. 

If we don't map this concept back to something the user is familiar with, they're not necessarily going to understand they already know what it is. You can see even in this short thread we are explaining ourselves using concrete examples (microscopy wavelengths, MRI T1 vs T2 images, satellite bands, etc.). I think that shows pretty clearly that when people try and explain a concept better, we don't just improve the definition, we add more examples to support it.